google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/StorageSnippets.java

```diff
@@ -179,6 +179,27 @@ public Blob createEncryptedBlob(String bucketName, String blobName, String encry
     return blob;
   }
 
+  /**
+   * Example of uploading a blob encrypted by a KMS-key.
+   */
+  // [TARGET create(BlobInfo, InputStream, BlobWriteOption...)]
+  // [VARIABLE "my_unique_bucket"]
+  // [VARIABLE "my_blob_name"]
+  // [VARIABLE "kms_key_name"]
+  public Blob createKmsEncrpytedBlob(String bucketName, String blobName, String kmsKeyName) {
```

Comment:
could you add similar variable documentation to this sample similar to setDefaultKmsKey